daemon/smartcard.c

```diff
@@ -784,15 +772,24 @@ smartcard_cb_container_change_pin(int fd, unsigned events, event_io_t *io, void
 			ERROR("Failed to receive message although EVENT_IO_READ was set. Aborting smartcard change_pin callback.");
 			event_remove_io(io);
 			event_io_free(io);
+			audit_log_event(container_get_uuid(startdata->container), FSA, CMLD,
+					CONTAINER_MGMT, "container-change-pin",
+					uuid_string(container_get_uuid(startdata->container)), 0);
 			control_send_message(CONTROL_RESPONSE_CONTAINER_CHANGE_PIN_FAILED, resp_fd);
 			return;
 		}
 		switch (msg->code) {
 		case TOKEN_TO_DAEMON__CODE__CHANGE_PIN_SUCCESSFUL: {
+			audit_log_event(container_get_uuid(startdata->container), SSA, CMLD,
+					CONTAINER_MGMT, "container-change-pin",
+					uuid_string(container_get_uuid(startdata->container)), 0);
 			control_send_message(CONTROL_RESPONSE_CONTAINER_CHANGE_PIN_SUCCESSFUL,
 					     resp_fd);
```

Comment:
remove whole block and just set the bool variable. (also in other case statements)
command_state = false;

Reply:
Done